### PR TITLE
Enhance pizza menu with descriptions and adjust layout

### PR DESCRIPTION
**Pull request by Mandla-tech**
With this PR we'll introduce the following changes to the pizza menu:

- Added descriptions to pizza items on the menu.
- Modified layout to display descriptions below pizza names.
- Updated styles for better readability and layout consistency.

-Adjusted forms layout and positioning for better view.

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -131,6 +139,7 @@ body {
   padding: 20px 10px;
   width: 80%;
   max-width: 600px;
+  transform: translateY(-8vh);
```
Comment (lawlawson) [inline styles/styles.css]: @Mandla-tech The 'transform: translate' is pushing the modal to high up on the screen and is potentially causing issues with the buttons which we saw on the call when screen sharing. Please can you remove this?

**Comment by lawlawson** [comment]
Screenshot 2024-07-27 at 00 33 37

There is too much white space at the top and bottom of this modal, this is a bad design principle and frowned upon. Please can you reduce the amount of spacing @Mandla-tech?

**Comment by Mandla-tech** [comment]
@lawlawson, the recommendations have been applied and also added responsive adjustments for smaller screens. The PR is ready for another review.

**Comment by lawlawson** [comment]
Great stuff @Mandla-tech I've looked locally and the changes look great!

**Review by lawlawson — APPROVED** [review]

**Review by lawlawson — APPROVED** [review]

**Pull request merged**